android/canonical/app/src/main/java/com/google/samples/quickstart/canonical/MapsFragment.kt

```diff
@@ -44,6 +44,9 @@ class MapsFragment : Fragment() {
     private lateinit var autocompleteFragment : AutocompleteSupportFragment
     private var currentLatLng : LatLng? = null
     private var targetMarker : Marker? = null
+    private val zoomValue = 14f
+    private val paddingRatio = 1.5
+    private val fragmentTag = "Mapfragment"
```

Comment:
These three values are compile-time constants. Could you put them in the companion object below with UPPER_CASE names?